### PR TITLE
Fix Clamav of Asset-manager

### PR DESCRIPTION
**Pull request by fredericfran-gds**
It is currently complaining about missing the clamd.conf so added it
from EC2 integration

### CODE REVIEW

File: charts/asset-manager/templates/worker-deployment.yaml [hunk 1]
```diff
@@ -69,6 +69,9 @@ spec:
           volumeMounts:
             - name: clam-virus-db
               mountPath: /var/lib/clamav
+            - name: {{ .Release.Name }}-clamd-conf
+              mountPath: /etc/clamav/clamd.conf
+              subPath: clamd.conf
```
Comment (sengi) [inline charts/asset-manager/templates/worker-deployment.yaml:74]: It's best not to use subPath because this prevents updating the configmap.

**Review by sengi — APPROVED** [review]

File: charts/asset-manager/templates/worker-deployment.yaml (hunk 1 above)
Comment (sengi) [reply charts/asset-manager/templates/worker-deployment.yaml:74]: I dunno to what extent we care in this case though. Not saying you need to change it, just something to be aware of when using it. If the dir is empty anyway apart from this file then I would get rid of the subPath and just mount as `/etc/clamav`, but if not then maybe it's not worth the faff.

File: charts/asset-manager/templates/worker-deployment.yaml [hunk 2]
```diff
@@ -69,6 +69,9 @@ spec:
           volumeMounts:
             - name: clam-virus-db
               mountPath: /var/lib/clamav
+            - name: {{ .Release.Name }}-clamd-conf
```
Comment (sengi) [inline charts/asset-manager/templates/worker-deployment.yaml:72]: No need for `Release.Name` here. The name is scoped to within the pod.

Comment (sengi) [inline charts/asset-manager/templates/worker-deployment.yaml:72]: Looks like you forgot to define this `Volume`?

**Review by sengi — APPROVED** [review]
Noticed a couple of other things

File: charts/asset-manager/templates/worker-deployment.yaml (hunk 1 above)
Comment (fredericfran-gds) [reply charts/asset-manager/templates/worker-deployment.yaml:74]: ok, let me see what's in the volume.

Comment (fredericfran-gds) [reply charts/asset-manager/templates/worker-deployment.yaml:74]: other stuff inside unfortunately.

File: charts/asset-manager/templates/worker-deployment.yaml (hunk 2 above)
Comment (fredericfran-gds) [reply charts/asset-manager/templates/worker-deployment.yaml]: I can fix that, thanks

**Pull request merged**